charts/auto-loans/origination-activity/figure-1a__number.html

```diff
@@ -0,0 +1,13 @@
+<html><head></head><body><div class="chart_wrapper" id="figure-1a__number" data-chart-type="line" data-source="num_data_AUT.csv" data-title="Number of loans originated" data-market="auto-loans" data-report-type="origination-activity">
+<style>@media only all and (max-width:37.4375em){}@media only all and (max-width:37.4375em){}@media only all and (max-width:37.5em){}.chart_wrapper{overflow-x:auto}.chart_wrapper .line{fill:none;stroke:steelblue;stroke-width:3px}.chart_wrapper .line__adjusted{stroke:#20aa3f}.chart_wrapper .line__unadjusted{stroke:#addc91}.chart_wrapper .line__projected{stroke-dasharray:4px,5px}.chart_wrapper .axis{font-family:'AvenirNextLTW01-Regular',Arial,sans-serif;font-style:normal;font-weight:normal;font-size:100%}.chart_wrapper .axis em,.chart_wrapper .axis i{font-family:'AvenirNextLTW01-Italic',Arial,sans-serif;font-style:italic;font-weight:normal}.lt-ie9 .chart_wrapper .axis em,.lt-ie9 .chart_wrapper .axis i{font-style:normal !important}.chart_wrapper .axis strong,.chart_wrapper .axis b{font-family:'AvenirNextLTW01-Demi',Arial,sans-serif;font-style:normal;font-weight:bold}.lt-ie9 .chart_wrapper .axis strong,.lt-ie9 .chart_wrapper .axis b{font-weight:normal !important}.chart_wrapper .axis__large{font-size:1.125em}.chart_wrapper .axis-label{font-family:'AvenirNextLTW01-Regular',Arial,sans-serif;font-style:normal;font-weight:normal;fill:#75787b;font-size:100%}.chart_wrapper .axis-label em,.chart_wrapper .axis-label i{font-family:'AvenirNextLTW01-Italic',Arial,sans-serif;font-style:italic;font-weight:normal}.lt-ie9 .chart_wrapper .axis-label em,.lt-ie9 .chart_wrapper .axis-label i{font-style:normal !important}.chart_wrapper .axis-label strong,.chart_wrapper .axis-label b{font-family:'AvenirNextLTW01-Demi',Arial,sans-serif;font-style:normal;font-weight:bold}.lt-ie9 .chart_wrapper .axis-label strong,.lt-ie9 .chart_wrapper .axis-label b{font-weight:normal !important}.chart_wrapper .axis-label__large{font-size:1.125em}.chart_wrapper .axis__x .domain{stroke:#d2d3d5}.chart_wrapper .axis__y .domain{stroke:transparent}.chart_wrapper .axis__projected .tick line{stroke-width:2px;stroke:#75787b}.chart_wrapper .axis__projected .tick text{fill:#75787b}.chart_wrapper .tick line{stroke:#d2d3d5}.chart_wrapper .tick text{fill:#75787b;font-size:100%}.chart_wrapper #bar text{font-size:200% !important}@media only all and (min-width:37.5625em){.chart_wrapper #bar text{font-size:100% !important}}</style>
+
+    <h3>Number of loans originated</h3>
```

Comment:
Referenced in a call we had: the legend is missing! :-o

I saw the extra spacing to accommodate it, but missed that it's because the legend was supposed to be there.